npm fails to publish releases

2.30.0 failed to publish:
```
npm notice
npm notice 📦  @deltachat/jsonrpc-client@2.30.0
npm notice Tarball Contents
npm notice 30B deltachat.ts
npm notice 73.3kB dist/deltachat.bundle.js
npm notice 66.0kB dist/deltachat.cjs
npm notice 30B dist/deltachat.d.ts
npm notice 67B dist/deltachat.js
npm notice 125B dist/deltachat.js.map
npm notice 49.0kB dist/generated/client.d.ts
npm notice 57.8kB dist/generated/client.js
npm notice 25.4kB dist/generated/client.js.map
npm notice 8.6kB dist/generated/constants.d.ts
npm notice 7.9kB dist/generated/constants.js
npm notice 5.7kB dist/generated/constants.js.map
npm notice 1.4kB dist/generated/jsonrpc.d.ts
npm notice 87B dist/generated/jsonrpc.js
npm notice 126B dist/generated/jsonrpc.js.map
npm notice 49.3kB dist/generated/types.d.ts
npm notice 85B dist/generated/types.js
npm notice 122B dist/generated/types.js.map
npm notice 1.9kB dist/src/client.d.ts
npm notice 2.8kB dist/src/client.js
npm notice 2.6kB dist/src/client.js.map
npm notice 252B dist/src/lib.d.ts
npm notice 283B dist/src/lib.js
npm notice 318B dist/src/lib.js.map
npm notice 0B generated/.gitkeep
npm notice 68.3kB generated/client.ts
npm notice 9.6kB generated/constants.ts
npm notice 1.4kB generated/jsonrpc.ts
npm notice 49.7kB generated/types.ts
npm notice 2.0kB package.json
npm notice 3.4kB src/client.ts
npm notice 252B src/lib.ts
npm notice 5.2kB test/basic.ts
npm notice 6.6kB test/online.ts
npm notice 2.2kB test/test_base.ts
npm notice 453B tsconfig.json
npm notice Tarball Details
npm notice name: @deltachat/jsonrpc-client
npm notice version: 2.30.0
npm notice filename: deltachat-jsonrpc-client-2.30.0.tgz
npm notice package size: 119.1 kB
npm notice unpacked size: 502.5 kB
npm notice shasum: 6132db914cdd90eb0298b18c322e700b31225042
npm notice integrity: sha512-aHb/3wPJSdWpg[...]6hfrLs86oizOQ==
npm notice total files: 36
npm notice
npm notice Publishing to https://registry.npmjs.org/ with tag latest and public access
npm notice publish Signed provenance statement with source and build information from GitHub Actions
npm notice publish Provenance statement published to transparency log: https://search.sigstore.dev/?logIndex=741352335
npm notice Access token expired or revoked. Please try logging in again.
npm error code E404
npm error 404 Not Found - PUT https://registry.npmjs.org/@deltachat%2fjsonrpc-client - Not found
npm error 404
npm error 404  '@deltachat/jsonrpc-client@2.30.0' is not in this registry.
npm error 404
npm error 404 Note that you can also install from a
npm error 404 tarball, folder, http url, or git url.
npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2025-12-04T17_40_26_075Z-debug-0.log
Error: Process completed with exit code 1.
```

This is since #7557 is merged.

--- a/.github/workflows/deltachat-rpc-server.yml
+++ b/.github/workflows/deltachat-rpc-server.yml
@@ -518,6 +518,11 @@ jobs:
           node-version: 20
           registry-url: "https://registry.npmjs.org"
 
+      # Ensure npm 11.5.1 or later is installed.
+      # It is needed for <https://docs.npmjs.com/trusted-publishers>
+      - name: Update npm
+        run: npm install -g npm@latest
+
       - name: Publish npm packets for prebuilds and `@deltachat/stdio-rpc-server`
         if: github.event_name == 'release'
         working-directory: deltachat-rpc-server/npm-package

--- a/.github/workflows/jsonrpc-client-npm-package.yml
+++ b/.github/workflows/jsonrpc-client-npm-package.yml
@@ -27,6 +27,11 @@ jobs:
           node-version: 20
           registry-url: "https://registry.npmjs.org"
 
+      # Ensure npm 11.5.1 or later is installed.
+      # It is needed for <https://docs.npmjs.com/trusted-publishers>
+      - name: Update npm
+        run: npm install -g npm@latest
+
       - name: Install dependencies without running scripts
         working-directory: deltachat-jsonrpc/typescript
         run: npm install --ignore-scripts

